packages/models-library/src/models_library/shared_user_preferences.py

```diff
@@ -0,0 +1,6 @@
+from .user_preferences import FrontendUserPreference
+
+
+class AllowMetricsCollectionFrontendUserPreference(FrontendUserPreference):
+    preference_identifier: str = "allowMetricsCollection"
```

Comment:
question, should this not be a literal? actually this should never ever change right?